jmclient/jmclient/configure.py

```diff
@@ -199,7 +199,7 @@ def jm_single():
 # running multiple bots at once. Don't alter it otherwise
 regtest_count = 0,0
 
-## SERVER 3/3) ILITA IRC (Tor - disabled by default)
+## SERVER: ILITA IRC (optional IRC alternate, Tor only)
 ################################################################################
 #[MESSAGING:server3]
```

Comment:
server3? server2 IRC server reliable [hackint](https://hackint.org/transport/tor#Transport__Tor) seems missing?

Reply:
OK, I'm pretty non-dogmatic about this part, and I think we can just settle on a setting when this merges. If you think darkscience, hackint and onion is the best choice that's fine with me. I think I am biased against hackint because it always seems to delay 30+ seconds before it lets you join the channel from when I'm watching it.

Reply:
Same here, joining Darkscience is much much quicker than Hackint.

Reply:
OK for now how about:

put the onion messaging section at the top.
then, put IRC SERVER 1, 2, 3 sections for darkscience, ilita, hackint, *with hackint commented out as a backup*. I'm suggesting it specifically because of the slowdown reason above.

That would be 1 onion + 2 IRC by default which seems logical to start us off with real-world usage until we get more confidence.

Reply:
I decided to do this right away in 
This is so we can do some final, as realistic as possible, testing. Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1182/commits/85e08c43f86cf81725ec0de0e776fb302fc653fb
Feel free to continue the convo if you don't think it's the best choice.